xblockmufi/private/edit.html

```diff
@@ -1,38 +1,93 @@
-<div class="wrapper-comp-settings is-active editor-with-buttons xblockmufi_edit" id="settings-tab">
-  <ul class="list-input settings-list">
-      <!-- TODO: Add an entry here for each field -->
-      <li class="field comp-setting-entry is-set">
-          <div class="wrapper-comp-setting">
-              <label
-                  class="label setting-label"
-                  for="xblock_xblockmufi_name"
-              >
-                  Name
-            </label>
-            <input
-                class="input setting-input"
-                id="xblock_xblockmufi_name"
-                value="{self.name}"
-                type="text"
-                tabindex="1"
-            >
-          </div>
-          <span
-              class="tip setting-help"
-          >
-              This name appears in the horizontal navigation at the top
-              of the page.
-          </span>
-      </li>
-  </ul>
-  <div class="xblock-actions">
-    <ul>
-      <li class="action-item">
-        <a href="#" tabindex="2" class="button action-primary save-button">Save</a>
-      </li>
-      <li class="action-item">
-        <a href="" tabindex="3" class="button cancel-button">Cancel</a>
-      </li>
+<div class="wrapper-comp-settings editor-with-buttons is-active" id="settings-tab">
+    <ul class="list-input settings-list">
```

Comment:
I was trying to figure out why the diff was so large here...

Did I really use 2-space indents in the SDK?! eww...

I'll have to fix that.